korge/src/commonMain/kotlin/com/soywiz/korge/view/Sprite.kt

```diff
@@ -0,0 +1,69 @@
+package me.emig.engineEmi.graphics.animationen
+
+import com.soywiz.klock.TimeSpan
+import com.soywiz.klock.measureTime
+import com.soywiz.klock.milliseconds
+import com.soywiz.korge.view.Image
+import com.soywiz.korio.async.delay
+
+class Sprite(vararg var animations: SpriteAnimation) : Image(animations[0].firstSprite) {
+    var stop = false
+    var currentAnimation : SpriteAnimation = animations[0]
```

Comment:
Some people might call `Sprite()` without arguments and get an exception here. Thinking on it, maybe we can deal 
 with the complexity of a set of animations in other place.
For example, instead of passing the id for playAnimation, passing the `SpriteAnimation` itself? With an initial one in the constructor? Each approach have its advantages and disadvantages. But I guess people could still have an animation collection or something that could be a separate class and then do something like (or just in plain local variables): `playAnimationForDuration(animation = animations["standing"])`. The Sprite class code is simplified (we only have to deal with one animation, and we know for sure it exists).
`animations` in the sample could be a simple map or a class like `SpriteAnimations` or `SpriteAnimationsCollection` handling the map.

Would be nice to know the use-cases of people. Maybe people want to just enable an animation looping forever. Maybe we can do something like (the play method is not suspend and uses the updater, thus reducing allocations and make it more stable in terms of performance/microstuttering):

```
val completedAnimation = Signal<Unit>()

init {
addUpdater { ms ->
   if (running) {
        // here take into account number of loops or something if wanted? I think there is a Times class that supports infinite loops and things like that
        // You can also trigger a signal when a loop of the animation completes so other methods can subscribe?
        // ... 
   }
}
}
```

With this approach we can do something like (not checked and incomplete):
* `fun play(animation) { this.animation = animation; this.frame = 0; running = true }` 
* `fun stop { running = false }`
* `suspend fun playAnimationForDuration() { running = true; delay(duration)  }`
* `suspend fun playAnimationTimes() { running = true; while (times.notZero) { completedAnimation.await()  ; times = times.prev } }`

It's late, so not too fresh and maybe I missed stuff. What do you think?